timemachine/cpp/src/smoothcore_nonbonded_interaction_group.hpp

```diff
@@ -0,0 +1,103 @@
+#pragma once
```

Comment:
Would it be desirable to also have a smoothcore implementation of `NonbondedPairListPrecomputed` for intramolecular nonbonded interactions?

Reply:
yes - but this is tricky. We'd need to do this *only* for all interactions outside of a dummy group, this can be all done in python though.